src/pkg/deps/deps.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dependencies.props))\dependencies.props" />
+
+  <PropertyGroup>
+    <TargetFramework>dnxcore50</TargetFramework>
```

Comment:
Dumb question, I'm sure, but why is this stuff dnxcore50 with this crazy fallback?  Can we not just say the targetframework is netstandard1.0?